cmd/neofs-node/config.go

```diff
@@ -969,8 +971,45 @@ func (c *cfg) configWatcher(ctx context.Context) {
 	}
 }
 
-// writeAppVersion writes app version as defined at compilation
-// step to the node's attributes.
-func writeAppVersion(c *cfg) {
+// writeSystemAttributes writes app version as defined at compilation
+// step to the node's attributes and an aggregated disk capacity
+// according to the all space on the all configured disks.
+func writeSystemAttributes(c *cfg) error {
+	// `Version` attribute
+
 	c.cfgNodeInfo.localInfo.SetAttribute("Version", misc.Version)
+
+	// `Capacity` attribute
+
+	var stat syscall.Statfs_t
+	var total uint64 // bytes
+	fsIDMap := make(map[syscall.Fsid]struct{})
+
+	for _, sh := range c.applicationConfiguration.EngineCfg.shards {
+		for _, storage := range sh.subStorages {
+			path := storage.path
+
+			err := util.MkdirAllX(path, storage.perm)
+			if err != nil {
+				return fmt.Errorf("can not create (ensure it exists) dir by '%s' path: %w", path, err)
+			}
+
+			err = syscall.Statfs(path, &stat)
+			if err != nil {
+				return fmt.Errorf("get FS stat by '%s' path: %w", path, err)
+			}
+
+			if _, handled := fsIDMap[stat.Fsid]; handled {
+				continue
+			}
+
+			total += stat.Blocks * uint64(stat.Bsize)
```

Comment:
definitely not
```suggestion
			total += stat.Bavail * uint64(stat.Bsize)
```
?

Reply:
tbh, i am not sure what [that attr](https://github.com/nspcc-dev/neofs-api/blob/953b4c5a9df1409da2a795b34bf9af66699689c6/netmap/types.proto#L160-L161) is expected to show. so you mean that every restart should change its attr value? i think that is about the total disk capacity to make a prediction about how huge a node is

/cc @roman-khimov

Reply:
It's going to be different with every node start. But at the same time you're correct in that if I'm a lazy mainnet admin, I don't care about shards, disks, whatever and just want my single FS on `/` to work then `blocks` will be somewhat inaccurate. Likely it's still good enough.

Reply:
i didn't think about restart. Attr may be changed anyway (e.g. node changed drives)

just confused by `Total available space ...` in protocol. Node has total space and node has available space. So `Capacity` may be misinterpreted, i'd clarify meaning in protocol first

and i didn't dive deeply in capacity issue, just drew attention. May be in such cases having 2 attrs (total and avail) is better than 1

Reply:
@roman-khimov, do not get it. after all, we need to announce available space (`stat.Bavail`) or the full disk capacity (`stat.Blocks`)?

that line was not a mistake, i printed disk capacity cause i thought that we need to print it

Reply:
`stat.Blocks`.